components/Global/Footer.jsx

```diff
@@ -0,0 +1,162 @@
+import {
+  AiOutlineMail,
+  AiOutlineInstagram,
+  AiOutlineTwitter,
+  AiFillLinkedin,
+  AiOutlineGithub
+} from 'react-icons/ai'
+import { FaDiscord } from 'react-icons/fa'
+const Footer = () => {
+  const settings = {
+    name: 'WebXDAO',
+    author: 'WebXDAO',
+    url: 'https://web3community.github.io'
+  }
+  const socials = [
+    {
+      name: 'Mail',
+      url: 'mailto:web3opensourcecommunity@gmail.com',
+      icon: AiOutlineMail,
+      text: 'Contact us via mail'
+    },
+    {
+      name: 'Instagram',
+      url: 'https://www.instagram.com/web3community/',
+      icon: AiOutlineInstagram,
+      text: 'Check our content at Instagram'
+    },
+    {
+      name: 'Twitter',
+      url: 'https://twitter.com/web3community',
+      icon: AiOutlineTwitter,
+      text: 'Follow us via Twitter'
+    },
+
+    {
+      name: 'Linkedin',
+      url: 'https://www.linkedin.com/company/web3community',
+      icon: AiFillLinkedin,
+      text: 'Connect with us at LinkedIn'
+    },
+    {
+      name: 'Twitter',
+      url: 'https://github.com/WebXDAO',
+      icon: AiOutlineGithub,
+      text: 'Contribute at GitHub'
+    },
+    {
+      name: 'Twitter',
+      url: 'https://discord.gg/TSRwqx4K2v',
+      icon: FaDiscord,
```

Comment:
Hmm, why would we use Fa for just a icon, maybe we could make it consistent?

Reply:
Hmm could not find discord icon in Ai
How about using fa for other icons as well ?